### ISSUE TITLE
Add Dark Mode / Theme Customization

### ISSUE THREAD

**Issue by 213sanjana** [eccentriccoder01/TalkHeal#8]
We should have a feature to allow users to switch between light and dark themes for improved comfort and accessibility. Also we can add a color palette theme selection which can sometimes make theme fresh and lighten up.
I think this should go in Level 2
@eccentriccoder01

**Comment by Resham011** [comment]
Hey,
I'd love to contribute to this! The idea of allowing users to switch between light and dark themes for better comfort and accessibility, along with color palette selections to freshen up the UI, sounds like a really valuable feature.
Please consider assigning this issue to me.
Thanks!

**Comment by eccentriccoder01** [comment]
@Resham011 I'm really happy to see your interest in contributing to TalkHeal❕But this issue has already been assigned before. If you believe you can improve this project meaningfully in unique and even better ways, you are always welcome to open new issues, work on unassigned ones, or wait for the assignees to complete their tasks.
